### ISSUE TITLE
_TypeError in ChromeProxyService._setUpChromeConsoleListeners.<anonymous closure>

### ISSUE THREAD

**Issue by christopherfujino** [dart-lang/webdev#2161]
Seeing this on flutter beta/3.12.0-1.1.pre with dwds 19.0.0:

```
_TypeError: type 'Null' is not a subtype of type 'String' in type cast
at ChromeProxyService._setUpChromeConsoleListeners.<anonymous closure>(chrome_proxy_service.dart:1111)
at _rootRunUnary(zone.dart:1407)
at _CustomZone.runUnary(zone.dart:1308)
at _CustomZone.runUnaryGuarded(zone.dart:1217)
at CastStreamSubscription._onData(async_cast.dart:85)
at _rootRunUnary(zone.dart:1407)
at _CustomZone.runUnary(zone.dart:1308)
at _CustomZone.runUnaryGuarded(zone.dart:1217)
at _BufferingStreamSubscription._sendData(stream_impl.dart:339)
at _BufferingStreamSubscription._add(stream_impl.dart:271)
at _SinkTransformerStreamSubscription._add(stream_transformers.dart:63)
at _EventSinkWrapper.add(stream_transformers.dart:13)
at WipDomain.eventStream.<anonymous closure>.<anonymous closure>(webkit_inspection_protocol.dart:377)
at _HandlerEventSink.add(stream_transformers.dart:209)
at _SinkTransformerStreamSubscription._handleData(stream_transformers.dart:111)
at _rootRunUnary(zone.dart:1407)
at _CustomZone.runUnary(zone.dart:1308)
at _CustomZone.runUnaryGuarded(zone.dart:1217)
at _BufferingStreamSubscription._sendData(stream_impl.dart:339)
at _DelayedData.perform(stream_impl.dart:515)
at _PendingEvents.handleNext(stream_impl.dart:620)
at _PendingEvents.schedule.<anonymous closure>(stream_impl.dart:591)
at _rootRun(zone.dart:1391)
at _CustomZone.run(zone.dart:1301)
at _CustomZone.runGuarded(zone.dart:1209)
at _CustomZone.bindCallbackGuarded.<anonymous closure>(zone.dart:1249)
at _rootRun(zone.dart:1399)
at _CustomZone.run(zone.dart:1301)
at _CustomZone.runGuarded(zone.dart:1209)
at _CustomZone.bindCallbackGuarded.<anonymous closure>(zone.dart:1249)
at _microtaskLoop(schedule_microtask.dart:40)
at _startMicrotaskLoop(schedule_microtask.dart:49)
at _runPendingImmediateCallback(isolate_patch.dart:123)
at _RawReceivePort._handleMessage(isolate_patch.dart:190)
```

Which for v19.0.0 is https://github.com/dart-lang/webdev/blob/e887316c2352524baaf3b2e5aadd15ba0773edf6/dwds/lib/src/services/chrome_proxy_service.dart#L1111

And at HEAD: https://github.com/dart-lang/webdev/blob/master/dwds/lib/src/services/chrome_proxy_service.dart#L1364

**Comment by aleliottasyn** [comment]
How long to merge the issue that solves the problem? If it takes that long I have to downgrade the flutter version :(

**Comment by aleliottasyn** [comment]
I see that the bug is fixed in 19.0.2, but flutter keeps downloading the buggy version(19.0.1+1). So I'm not able to update my app. Please let me figure out how to fix this. @elliette

**Comment by christopherfujino** [comment]
> I see that the bug is fixed in 19.0.2, but flutter keeps downloading the buggy version(19.0.1+1). So I'm not able to update my app. Please let me figure out how to fix this. @elliette

@aleliottasyn this fix is available in Flutter beta 3.14.0-0.2.pre, I would recommend switching to that release in order to unblock the debugging workflow. Given the current impact of this bug, and the cost of releasing hotfixes, I don't think we will be considering hotfixing this at this time.
